tripy/tests/integration/test_gather.py

```diff
@@ -0,0 +1,43 @@
+#
+# SPDX-FileCopyrightText: Copyright (c) 1993-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import cupy as cp
+import numpy as np
+import pytest
+
+import tripy as tp
+
+
+class TestGatherOp:
+    @pytest.mark.parametrize(
+        "x_shape, axis, indices",
+        [
+            ((2, 3), 0, (1)),
+            pytest.param((2, 3, 4), 1, (1, 2),marks=pytest.mark.xfail(reason="Bug in gather op. (Remove xfail when fixed)")),
```

Comment:
What's the bug? Can you file an issue with details? (1 issue for all failures)

Reply:
I filed an issue on gitlab should I file one on github instead?

Reply:
Github.